src/commands/retention-policies/create.js

```diff
@@ -87,6 +93,30 @@ RetentionPoliciesCreateCommand.flags = {
 			'remove_retention'
 		]
 	}),
+	description: flags.string({
+		required: false,
+		description: 'The additional text description of the retention policy'
+	}),
+	'can-owner-extend-retention': flags.boolean({
+		description: 'The owner of a file will be allowed to extend the retention',
+		allowNo: true
+	}),
+	'custom-notification-recipients': flags.string({
```

Comment:
Shouldn't we use a singular form here as we can only add one receipient per entry?
`custom-notification-recipients` => `custom-notification-recipient`